docs/testplan.md

```diff
@@ -564,24 +564,35 @@ authentication:
 - [ ] Change `second_factor` type to `on` and verify that mfa is required (no option `none` in dropdown)
 
 #### MFA require auth
-Through the CLI, `tsh login` and register a WebAuthn key with `tsh mfa add` (not supported in UI yet).
+Through the CLI, `tsh login` and register a WebAuthn key with `tsh mfa add`.
 
 Using the same user as above:
 - [ ] Verify logging in with registered WebAuthn key works
 - [ ] Verify connecting to a ssh node prompts you to tap your registered WebAuthn key
 - [ ] Verify in the web terminal, you can scp upload/download files
 
-#### MFA Management
-TODO(yassine)
+#### MFA Management (UI)
```

Comment:
```suggestion
#### MFA Management
```

this section (and below) is actually under a huge section called [Web UI](https://github.com/gravitational/teleport/blob/master/docs/testplan.md#web-ui), so all test under it implies it's for UI